docs/src/infra/privilege-escalation/windows/weak-service-permissions.md

```diff
@@ -1,2 +1,129 @@
+
+Here's a quick summary of Windows file permissions based on the provided table:
+
+    F (Full access): Grants complete control over the file, including modifying and deleting it.
+    M (Modify access): Allows editing and deleting the file, but not changing permissions.
+    RX (Read and execute access): Permits viewing the file and executing it if it's an executable.
+    R (Read-only access): Only allows viewing the file without making changes.
+    W (Write-only access): Permits writing to the file but not reading or executing it.
+
+## Pratice
+
+### Insecure Permissions on Service Executable :
+
+This type of abuse can occur when the binary of a Windows service is misconfigured, meaning that the permissions for this binary allow, for example, a local user without special privileges to modify it or even replace it with a malicious one, the Attacker can obtain the privileges of the service account in general a high privileges account.
+
+For the enumeration can be done with Powershell, first using native Powershell commands (Living off the Land) But also with [PowerUp](https://github.com/PowerShellMafia/PowerSploit/blob/master/Privesc/PowerUp.ps1). The main difference between the two is on the result PowerUp is able to tell you if a service is vulnerable or not while PowerShell just lists the Binary Services. In this example the **aniService** service is misconfigured and vulnerable to this attack.
+
+```powershell
+# USING POWERSHELL
+> Get-CimInstance -ClassName win32_service | Select Name,State,PathName | Where-Object {$_.State -like 'Running'}
+
+# USING POWERUP
+> Invoke-AllChecks
+[*] Checking service executable and argument permissions...
+
+
+ServiceName    : aniService
+Path           : C:\Program Files\AniService\ani.exe
+ModifiableFile : C:\Program Files\AniService\ani.exe
+StartName      : LocalSystem
+AbuseFunction  : Install-ServiceBinary -ServiceName 'aniService'
+```
+
+If you have identified your vulnerable binary with a tool like PowerUp or even Winpeas, you can check the permissions of this binary with **icacls.exe**. The most interesting permissions are (F) and (M) if they are related to a group such as `Everyone:` or a local user for exemple. 
+```powershell
+> icacls "C:\Program Files\AniService\ani.exe"
+C:\Program Files\AniService\ani.exe AUTORITÉ DE PACKAGE D’APPLICATION\TOUS LES PACKAGES D’APPLICATION:(RX)
+                                    AUTORITÉ DE PACKAGE D’APPLICATION\TOUS LES PACKAGES D’APPLICATION RESTREINTS:(RX)
+                                    AUTORITE NT\Système:(F)
+                                    BUILTIN\Administrateurs:(F)
+                                    Tout le monde:(F)
+                                    BUILTIN\Utilisateurs:(RX)
```

Comment:
Can you see to have this type of content be in English for our non-French readers please 🙏 ?